tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Hmm. We'll need to add `import 'typescript/lib/lib.dom';` to any files in these packages using fetch types.

Reply:
I'm not sure if thats the right way to do that. That may import dom types globally for all files. As in it may pollute the files of the user's files that are depending on `airtable.js`.